### ISSUE TITLE
Notary dealing with investors that are not cleared. 

### ISSUE THREAD

**Issue by ilhanu** [Fair-Squares/fair-squares#198]
f there is something found about on or more investors when the housing is in `finalizing` state, the housing status should be demoted to `Onboarded` and the investors should get their allocation unreserved. So there will be a new round. But what do we do with he the remaining investors that are in question ?  

This would mean that the investor that is found to be fraudulent/suspicious according to the notary because of outstanding debt or something else would not be able to participate in the next round. So we would need to give investors also a status. [Verified, Under-Review, Rejected] ?

**Comment by ndkazu** [comment]
I think a new status & waiting list is needed for these investors if we still want to give them a chance to stay in the investor pool after the Notary investigation.
If the Notary investigation is more than enough, we only need to kick them out of the pool.

**Comment by ilhanu** [comment]
Agreed, also decide what do with them for new selection. So if the bidding pallet queries the on-chain states again, it should come up with the same investors + the new ones that are taking the place of the ones that have been left out and marked by the notary. 

A better and harder design would be that the bidding pallet kicks just the investors out that are marked and replaces them with new investors, so the reserved candidates don't get unreserved.

**Comment by letodunc** [comment]
Why do we need this check on investors by the notary as they have already contributed in the fund? Maybe here a problem can be detected concerning the owner. The entity buying the house is the fund that have the needed amount for the house. Those checks on investors has been done when they contribute to the fund I think.
